ERCS/erc-7895.md

```diff
@@ -0,0 +1,230 @@
+---
+eip: 7895
+title: API for Hierarchical Accounts
+description: Adds JSON-RPC method for requesting that a universal wallet create or track another account that it owns
+author: Wilson Cusack (@wilsoncusack), Jake Feldman (@jakefeldman), Montana Wong (@montycheese), Felix Zhang (@fan-zhang-sv), Jake Moxey (@jxom)
+discussions-to: https://ethereum-magicians.org/t/wallet-addsubaccount/23013
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-02-18
+---
+
+## Abstract
+
+This ERC introduces a new wallet RPC, `wallet_addSubAccount`, which allows an app to request a wallet track a smart account that the wallet owns. It also allows apps to request the wallet to provision a new account, owned by the universal wallet with a signer provided by the caller. 
```

Comment:
I still think you need more details about what "track" means in this context. I assume that the idea is that the dapp wants a unique address, and "track" means both "display in the wallet UI" and "allow the user to send transactions from it"?